.github/workflows/test.yml

```diff
@@ -16,6 +16,6 @@ jobs:
     - name: Checkout code
       uses: actions/checkout@v2
     - name: Build
-      run: make BUILD_IN_CONTAINER=false cmd/agent/agent cmd/agentctl/agentctl cmd/agent-operator/agent-operator
+      run: make GOFLAGS="-tags=netgo,nodocker" BUILD_IN_CONTAINER=false cmd/agent/agent cmd/agentctl/agentctl cmd/agent-operator/agent-operator
```

Comment:
do we need netgo here since it's already defined in the Makefile where this GOFLAGS variable is expanded?

Reply:
Yeah unfortunately, passing GOFLAGS here overrides the entire set of tags so you need to give netgo again